stl/inc/concepts

```diff
@@ -32,7 +32,7 @@ _STD_BEGIN
 template <class _Ty1, class _Ty2>
 concept _Same_impl = // Must be a distinct concept to provide symmetric subsumption for same_as
 #ifdef __clang__
-    __is_same(_Ty1 _Ty2);

```

Comment:
That worked before ?! Wow!

Reply:
No, it never did: there was once a comma here, which was inadvertently deleted. We never noticed due to the lack of clang concepts test coverage.

Reply:
OIC